web/src/components/refresh-dropdown.tsx

```diff
@@ -48,7 +48,7 @@ export const RefreshDropdown: React.FC<Props> = ({ id, interval, setInterval })
       onSelect={() => setIsOpen(false)}
       toggle={
         <DropdownToggle id={`${id}-dropdown`} onToggle={() => setIsOpen(!isOpen)}>
-          {refreshOptions[selectedKey]}
+          {refreshOptions[selectedKey as keyof typeof refreshOptions]}
```

Comment:
This is not related to this PR but the `as keyof typeof` cast made me think about this : 
What should we do if the key is not in the options (like change the url parameter) ?
- fallback on default value
- allow custom value and show `key` instead

Reply:
actually, it looks like refresh isn't stored in URL, i'm going to add it (unless it was not done on purpose?)

Reply:
to (not) answer your question, i don't really know what's best. We could also show something like "url-defined" on the dropdown in that particular situation? That's a very corner case, no?

Reply:
Ho yes you are right, it's only for `time-range` not `refresh`.

All non filters parameters are supposed to be in [localStorage](https://issues.redhat.com/browse/NETOBSERV-125) so you can keep it as is for now.

Reply:
But it could also be changed in the localStorage :angel: